Formula/modsman.rb

```diff
@@ -0,0 +1,29 @@
+class Modsman < Formula
+  desc "Minecraft mod manager for the command-line"
+  homepage "https://gitlab.com/sargunv-mc-mods/modsman"
+  url "https://gitlab.com/sargunv-mc-mods/modsman/-/archive/0.20.1/modsman-0.20.1.tar.gz"
```

Comment:
```ruby
  url "https://gitlab.com/sargunv-mc-mods/modsman.git",
      :tag      => "0.20.1",
      :revision => "b23331f7080b5a2718ab43e4120372bcf3421b6d"
```

Reply:
You do not need to create a fake repository with this.

Reply:
Ah, nice, that's what I was looking for. Will fix.